src/unleash-proxy.ts

```diff
@@ -206,17 +208,24 @@ export default class UnleashProxy {
         }
     }
 
-    lookupToggles(req: Request, res: Response<FeaturesSchema | string>): void {
+    lookupToggles(
+        req: Request<any, any, LookupTogglesSchema>,
+        res: Response<FeaturesSchema | string>,
+    ): void {
         const clientToken = req.header(this.clientKeysHeaderName);
 
         if (!this.ready) {
             res.status(503).send(NOT_READY_MSG);
         } else if (!clientToken || !this.clientKeys.includes(clientToken)) {
             res.sendStatus(401);
         } else {
-            const { context, toggles: toggleNames = [] } = req.body;
+            const { context = {}, toggles: toggleNames = [] } = req.body;
+
+            const toggles = this.client.getDefinedToggles(
+                toggleNames,
+                context as Context,
```

Comment:
Because I can't make OpenAPI (or the packages we use) recognize the typing correctly, this seems to be the only way to make it accept the context. The OpenAPI integration in this project is less mature than the one in unleash/unleash, so it may just be things that we haven't configured yet. But still, I think this cast is plenty acceptabel.

Reply:
Agreed!